### PR TITLE
feat: landing page & result page

### PR DESCRIPTION
**Pull request by bhtibrewal**
Landing page
- categories section
image
- Leaderboard
image

result page
-score
- answers
image
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,143 @@
...
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/617c60ee1d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="https://artsyui.netlify.app/src/artsyUI.css" />
```
Comment (Ankur9669) [inline index.html:7]: You should add the script tags in bottom

File: index.html [hunk 2]
```diff
@@ -1,12 +1,143 @@
...
+                    <div class="leader-rank">#1</div>
+                    <div class="leader-img">
+                        <img alt="leader avatar" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQGtnn5CQr_nEzmp2EDlJAvgXaA4n1taH-sDQ&usqp=CAU">
+                    </div>
+                    <div class="flex-col leader-name">
+                        <div><strong>Avenger</strong></div>
+                        <div>Bhavika Tibrewal</div>
+                    </div>
+                    <div class="body-l leader-points primary">300pts</div>
+                </div>
```
Comment (Ankur9669) [inline index.html]: 300pts is a text, so you can replace div with a p tag which i think would be more accesible

File: index.html [hunk 3]
```diff
@@ -1,12 +1,143 @@
...
+          <h1 class="section-heading">Leaderboard</h1>
+            <div class="leader-list">
+                <div class="container leader-sec flex-align-center">
+                    <div class="leader-rank">#1</div>
+                    <div class="leader-img">
+                        <img alt="leader avatar" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQGtnn5CQr_nEzmp2EDlJAvgXaA4n1taH-sDQ&usqp=CAU">
+                    </div>
+                    <div class="flex-col leader-name">
+                        <div><strong>Avenger</strong></div>
+                        <div>Bhavika Tibrewal</div>
```
Comment (Ankur9669) [inline index.html]: Same here also you could use p tags

File: index.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    rootDoc.style.setProperty("--light-bg", "#fcfcfc");
+}
+modeBtn.addEventListener("click", () => {
+    console.log(modeBtn.className);
+    if (modeBtn.classList.contains("fa-moon")) {
+        darkmode();
+    }
+    else {
+        lightmode();
+    }
```
Comment (Ankur9669) [inline index.js:27]: Could remove the console.log statement
And you could use ternary operator instead of if else

File: pages/result/result.html [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/617c60ee1d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="https://artsyui.netlify.app/src/artsyUI.css" />
```
Comment (Ankur9669) [inline pages/result/result.html:7]: You can add script tags at bottom

File: pages/result/result.html [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+        <h2>Well Played</h2>
+      </section>
+      <section class="container score-card flex-col">
+        <p class="body-l"><i class="fa-solid fa-star fa-xs primary"></i>Score</p>
+        <h2 class="primary">300</h2>
+        <div class="flex-align-center score-sec">
+          <div class="flex-col score-col">
+            <span>Total</span>
+            <div>7</div>
+          </div>
```
Comment (Ankur9669) [inline pages/result/result.html:35]: You could use p tag for displaying numbers instead of div

File: pages/result/result.html [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+            <span><i class="fa-solid fa-xmark"></i>Wrong</span>
+            <div>2</div>
+          </div>
+        </div>
+      </section>
+      <section class="section">
+        <div class="question-sec flex-col">
+          <p class="body-l">
+              <div><strong>Question 1:</strong></div>
+             In The Force Awakens, which character has Darth Vader’s damaged
```
Comment (Ankur9669) [inline pages/result/result.html:50]: Here also you can replace div

File: pages/result/result.html [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+            mask?
+          </p>
+          <div class="options">
+            <div class="fa-solid right-op">
```
Comment (Ankur9669) [inline pages/result/result.html:54]: Div is also fine but you can also use the unordered list here

File: style.css [hunk 9]
```diff
@@ -0,0 +1,83 @@
+:root {
+  --quiz-bg: #dfe2eb;
+  --quiz-bg-dark: #272a2c;
+  --quiz-header: #0e131f;
+  /* --primary-color: rgb(204, 31, 31); */
+  --wrong-option: red;
+  --right-option: green;
+}
```
Comment (Ankur9669) [inline style.css:8]: You can use hex here to maintain consistency

File: style.css [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+  overflow: scroll;
+}
+main::--webkit-scrollbar {
+  width: 0.5rem;
+  height: 1rem;
+}
+main::--webkit-scrollbar-thumb {
+  background: var(--primary-color);
+}
+.quiz-footer {
```
Comment (Ankur9669) [inline style.css:55]: Add the comment for the webkit scrollbar css, so that the reviewer could understand that it is for hiding the scrollbar

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
image
The text right and wrong are not vertically aligned

image
In the responsive mode you texts are breaking
And you can add a little margin beetween question heading and question in mobile view

The code is so good that i have to nitpick things in the code😂. So really great work!!

File: index.html (hunk 2 above)
Comment (bhtibrewal) [reply index.html]: Definately that should have been a p tag, will change

File: index.js (hunk 4 above)
Comment (bhtibrewal) [reply index.js:27]: I have not used ternary to keep scope of expansion open

File: pages/result/result.html (hunk 5 above)
Comment (bhtibrewal) [reply pages/result/result.html:7]: Okay will change

File: style.css (hunk 10 above)
Comment (bhtibrewal) [reply style.css:55]: Okay will add

File: README.md [hunk 11]
```diff
@@ -0,0 +1,7 @@
+# StarBIZ Quiz
+- Landing Page 
+- Categories for quiz
+https://user-images.githubusercontent.com/42600164/154273139-0ab786da-23d5-4d29-a654-a2e037744198.png
+https://user-images.githubusercontent.com/42600164/154273268-403f604a-43b5-4fe1-9c7b-90761ad28824.png
+https://user-images.githubusercontent.com/42600164/154273580-1fc1baf8-d6e4-43d0-b2cc-70216ff33bf6.png
+https://user-images.githubusercontent.com/42600164/154273627-94128a20-f21b-4110-86b8-ceb02892ede4.png
```
Comment (iRohitGaur) [inline README.md:7]: don’t put png file names here

File: homepage.css [hunk 12]
```diff
@@ -0,0 +1,53 @@
+.categories-grid {
+  gap: 5%;
+}
+.categories-sec .card {
+  border: none;
+  transition: none;
+  transition: transform 1s ease-out;
```
Comment (iRohitGaur) [inline homepage.css]: transition: none; is not needed here.

File: homepage.css [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+}
+.categories-sec .card:hover {
+  transform: scale(1.1);
+}
+
+/* leader section */
+.leaderboard-sec {
+  align-items: stretch;
+  max-width:1000px;
```
Comment (iRohitGaur) [inline homepage.css]: use relative units

File: homepage.css [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+  margin:auto;
+}
+.section-heading {
+  align-self: center;
+}
+.leader-list{
+    gap:2rem;
+}
+.leader-sec{
+    transition:transform 0.5s ease-out;
```
Comment (iRohitGaur) [inline homepage.css]: indentation issues

File: index.html (hunk 1 above)
Comment (iRohitGaur) [reply index.html:7]: +1

File: index.html [hunk 15]
```diff
@@ -1,12 +1,143 @@
...
+    <title>StarBiz</title>
+  </head>
+  <body class="quiz-body">
+    <header class="quiz-header flex-align-center">
+      <a href="#"><div class="logo">StarBiz</div></a>
+      <div class="right-side flex-align-center">
+        <i class="fab fa-github"></i>
+        <i class="far fa-user"></i>
+        <i class="fas fa-moon mode-btn"></i>
+        <!-- <i class="fas fa-sun"></i> -->
```
Comment (iRohitGaur) [inline index.html:19]: remove commented code

File: index.html [hunk 16]
```diff
@@ -1,12 +1,143 @@
...
+    <!-- <footer class="quiz-footer flex-col">
+        <p>Made with 
+            <i class="fas fa-heart"></i> 
+            by <span class="name">Bhavika Tibrewal</span></p>
+        <div>
+            <i class="fab fa-github"></i>
+            <i class="fab fa-twitter"></i>
+            <i class="fab fa-linkedin"></i>
+        </div>
+    </footer> -->
```
Comment (iRohitGaur) [inline index.html:140]: remove commented code

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge with changes 👍🏻

File: index.js (hunk 4 above)
Comment (Ankur9669) [reply index.js:27]: Understood

**Pull request merged**